queue_services/entity-filer/src/entity_filer/filing_processors/continuation_out.py

```diff
@@ -32,7 +32,6 @@ def process(business: Business, continuation_out_filing: Filing, filing: Dict, f
 
     continuation_out_json = filing['continuationOut']
 
-    details = continuation_out_json.get('details')
```

Comment:
Remove this code

![image](https://github.com/bcgov/lear/assets/64869009/d9e8702e-6803-423a-983d-b0e9b836a790)